<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="TWizardPage" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>305</height>
   </size>
  </property>
  <property name="title">
   <string>Register with server</string>
  </property>
  <layout class="QVBoxLayout" name="registerPageVerticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="centralLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="subtitleLabel">
       <property name="text">
        <string>A &lt;b&gt;keyfile&lt;/b&gt; is used to encrypt your data before uploading.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TAsideLabel" name="tasidelabel">
       <property name="text">
        <string>Don't have an account? Register on &lt;a href=&quot;http://tarsnap.com&quot;&gt;tarsnap.com&lt;/a&gt;.</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="keyfileTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>133</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>133</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::tab-bar { alignment: center; }</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="createKeyfileTab">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Create new keyfile</string>
        </attribute>
        <layout class="QFormLayout" name="createKeyfileLayout">
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="registerPageInfoLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>17</height>
            </size>
           </property>
           <property name="text">
            <string>Choose a machine name and enter your account credentials.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignJustify|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="machineNameLabel">
           <property name="text">
            <string>Machine name:</string>
           </property>
           <property name="buddy">
            <cstring>machineNameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="machineNameLineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>this machine's name</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="tarsnapUserLabel">
           <property name="text">
            <string>Tarsnap user:</string>
           </property>
           <property name="buddy">
            <cstring>tarsnapUserLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="tarsnapUserLineEdit">
           <property name="placeholderText">
            <string>your e-mail</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="tarsnapPasswordLabel">
           <property name="text">
            <string>Tarsnap password:</string>
           </property>
           <property name="buddy">
            <cstring>tarsnapPasswordLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="tarsnapPasswordLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>your password</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="useExistingKeyfileTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>104</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Use existing keyfile</string>
        </attribute>
        <layout class="QVBoxLayout" name="useExistingKeyfileLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="TPathComboBrowse" name="keyfilePathComboBrowse">
           <property name="label">
            <string>Select your existing keyfile:</string>
           </property>
           <property name="placeholderText">
            <string>this machine's keyfile</string>
           </property>
           <property name="dialogTitle">
            <string>Browse for existing machine key</string>
           </property>
           <property name="dialogFilter">
            <string>Tarsnap key files (*.key *.keys)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="useExistingKeyfileSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="TProgressWidget" name="progressWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="footerLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="footerLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="footerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Register Machine</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TAsideLabel</class>
   <extends>QLabel</extends>
   <header>TAsideLabel.h</header>
  </customwidget>
  <customwidget>
   <class>TPathComboBrowse</class>
   <extends>QWidget</extends>
   <header>TPathComboBrowse.h</header>
  </customwidget>
  <customwidget>
   <class>TProgressWidget</class>
   <extends>QWidget</extends>
   <header>TProgressWidget.h</header>
  </customwidget>
  <customwidget>
   <class>TWizardPage</class>
   <extends>QWidget</extends>
   <header>TWizardPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nextButton</tabstop>
  <tabstop>keyfileTabWidget</tabstop>
  <tabstop>machineNameLineEdit</tabstop>
  <tabstop>tarsnapUserLineEdit</tabstop>
  <tabstop>tarsnapPasswordLineEdit</tabstop>
  <tabstop>keyfilePathComboBrowse</tabstop>
  <tabstop>backButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
